Eastern-Research-Group/csb-rebate-forms-app
# Update 2024 NCES JSON file

## Related Issues:
* CSBAPP-441

## Main Changes:
* Another follow-up to #483 – Update 2024 NCES JSON file again – this time remove existing blank "A999999" record, and update existing "A999990" record to "A999999".

## Steps To Test:
1. Navigate to a 2024 FRF submission, and advance to the "School District Info" page
2. Type in "A999990" as the NCES District ID – you should now see no results (was previously Morongo School).
3. Type in "A999999" as the NCES District ID – you should now see one result for Morongo School.

## TODO:
- [ ] Confirm the 2024 FRF definition has been updated to remove the A999999 logic that unlocks school district fields from being read-only (**NOTE:** that work is done within the Formio form definition itself – completely independent of the wrapping application).
